Source/Visitable/VisitableView.swift

```diff
@@ -68,6 +69,16 @@ open class VisitableView: UIView {
         
         #if !targetEnvironment(macCatalyst)
         scrollView.addSubview(refreshControl)
+
+        /// Infer refresh control's default height from its frame, if given.
+        /// Otherwise fallback to 60 (the default height).
+        let refreshControlHeight = refreshControl.frame.height > 0 ? refreshControl.frame.height : 60
+        
```

Comment:
Why do we have to specify the height?
Internally, refresh control doesn't use autolayout for positioning its subviews. Its main frame doesn't reflect its true size. The main container height is 60pt, but the view where the refresh image/animation is has a height of 100pt. When using autolayout the latter is used. 🤷‍♂️ Even if we don't set the height, the refresh control will respect the safe area, but it won't be positioned as it should be.

<img width="1920" alt="Screenshot 2023-05-17 at 11 52 53" src="https://github.com/hotwired/turbo-ios/assets/617660/49e2976b-5aa8-4350-b285-1162c2d38f86">